src/App.js

```diff
@@ -0,0 +1,23 @@
+import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+
+import Categories from './component/categories';
+import Navbar from './component/Navbar/NavBar';
+import Books from './component/Books/Books';
+
+function App() {
+  return (
+    <div className="App">
+      <Router>
+        <Navbar />
```

Comment:
Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 😎